readthedocs/search/api.py

```diff
@@ -48,27 +47,13 @@ def get_inner_hits(self, obj):
             sections = inner_hits.sections or []
             domains = inner_hits.domains or []
             all_results = itertools.chain(sections, domains)
-
-            sorted_results = [
-                {
-                    'type': hit._nested.field,
-                    '_source': hit._source.to_dict(),
-                    'highlight': self._get_inner_hits_highlights(hit),
-                }
-                for hit in sorted(all_results, key=attrgetter('_score'), reverse=True)
-            ]
-
+            sorted_results = list(utils._get_sorted_results(
+                results=all_results,
+                source_key='_source',
+                logging=True,
```

Comment:
optional logging feels weird. Do we ever not want logs?

Reply:
I think we should probably just log outside this function, and remove the logging from this function call.

Reply:
Done.